flink-streaming-java/src/main/java/org/apache/flink/streaming/api/environment/StreamExecutionEnvironment.java

```diff
@@ -277,6 +277,29 @@ public StreamExecutionEnvironment enableCheckpointing(long interval) {
 	/**
 	 * Enables checkpointing for the streaming job. The distributed state of the streaming
 	 * dataflow will be periodically snapshotted. In case of a failure, the streaming
+	 * dataflow will be restarted from the latest completed checkpoint. This method selects
+	 * {@link CheckpointingMode#EXACTLY_ONCE} guarantees.
+	 *
+	 * <p>The job draws checkpoints periodically, in the given interval. The state will be
+	 * stored in the configured state backend.</p>
+	 *
+	 * <p>NOTE: Checkpointing iterative streaming dataflows in not properly supported at
+	 * the moment. For that reason, iterative jobs will not be started if used
+	 * with enabled checkpointing. To override this mechanism, use the 
+	 * {@link #enableCheckpointing(long, CheckpointingMode, boolean)} method.</p>
+	 *
+	 * @param interval Time interval between state checkpoints in milliseconds.
+	 * @param minInterval Time interval between the last finished checkpoint and the next one in mills.
+	 */
+	public StreamExecutionEnvironment enableCheckpointing(long interval, long minInterval) {
```

Comment:
`minInterval` might be confusing for users because it actually is the pause between successive checkpoints.